yt_dlp/extractor/radiko.py

```diff
@@ -207,9 +209,10 @@ def _real_extract(self, url):
                     'ft': radio_begin,
                     'end_at': radio_end,
                     'to': radio_end,
-                    'seek': video_id,
+                    'seek': timestring,
                 },
             ),
+            '_old_archive_ids': [timestring],
```

Comment:
normally we'd wanna do this, but since these are livestreams I think there is no benefit... it could only cause problems if someone is trying to download two different streams with the same `timestring`

(btw it would be `'_old_archive_ids': [make_archive_id(self, timestring)]`)

```suggestion
```